public/index.html

```diff
@@ -25,6 +25,7 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>React App</title>
+    <link href="https://fonts.googleapis.com/css2?family=Space+Mono&family=Work+Sans:wght@400;500;700&display=swap" rel="stylesheet">
```

Comment:
@Jeff-Enriquez do we want to add fonts according to [this](https://stackoverflow.com/a/41678350/7897413)

Reply:
That way would be used if we were loading the fonts locally. It's recommended to load it in the html but I came across something that will improve performance. This (link)[https://scotch.io/@micwanyoike/how-to-add-fonts-to-a-react-project] covers all three methods.